src/zed_lines.mli

```diff
@@ -74,3 +74,7 @@ val replace : t -> int -> int -> t -> t
   (** [replace set offset length repl] replaces the subset at offset
       [offset] and length [length] by [repl] in [set]. *)
 
+val get_idx_by_width : t -> int -> int -> int
+  (** [get_idx_by_width set column_width] return the offset of the char at
```

Comment:
The spec in the documentation comment looks wrong; it has only two arguments while the function takes 3

Reply:
Thank you, doc comment is fixed now.